openedx/features/edly/utils.py

```diff
@@ -333,3 +334,47 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context_data(default_html_certificate_configuration):
+    """
+    Gets current site's context data for invalid certificate.
+
+    Try to get current site's context data for invalid certificate from site configuration
+    or fallback to default html certificate configurations.
+
+    Arguments:
+        default_html_certificate_configuration (dict): Default html configurations dict.
+
+    Returns:
+        dict: Context data.
+    """
+    context = {}
+
+    for key, value in default_html_certificate_configuration.get('default').items():
```

Comment:
why not initialize context with `dict`? like:
```
    context = dict(default_html_certificate_configuration.get('default'))
```

Reply:
Done